.github/workflows/playwright.yml

```
@@ -144,9 +147,64 @@ jobs:
+          include-hidden-files: true
+
+      - name: Generate Coverage Report
+        run: cd tests/ui-testing && npx nyc report
+
       - uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: playwright-report
-          path: playwright-report/
-          retention-days: 30
+            name: playwright-report
```

yamllint

[warning] wrong indentation: expected 10 but found 12

(indentation)